autrainer/transforms/specific_transforms.py

```diff
@@ -722,3 +720,41 @@ def __init__(
     def __call__(self, data: torch.Tensor) -> torch.Tensor:
         data = self.smile.process_signal(data.numpy(), self.sample_rate)
         return torch.from_numpy(self.smile.to_numpy(data)).squeeze()
+
+
+class PadCollate(AbstractTransform):
```

Comment:
How is that different from [RandomCrop](https://audtorch.readthedocs.io/en/0.2.0/api-transforms.html#randomcrop) which is already available?

Reply:
Also, the problem with `RandomCrop` is that you also have to adapt the `labels`. How do you handle that here?